### ISSUE TITLE
What is Aurababeous?

### ISSUE THREAD

**Issue by lsaether** [w3f/polkadot-wiki-old#240]
- [ ] Document the block production known as Aurababeous.

Please see https://github.com/paritytech/substrate/pull/3380 for context and information. Maybe this could land as a paragraph in the Babe [section](https://wiki.polkadot.network/en/latest/polkadot/learn/consensus/#babe-block-production) on the consensus page. Although I think that page is due to an overhaul and we should extract Babe to its own page on the wiki.

**Comment by andresilva** [comment]
Just my 2 cents but imo we should keep the BABE name and just extend the definition with primary (VRF based) and secondary (slot number based) slots. The secondary slot assignment logic isn't really related to Aura anyway.

**Comment by burdges** [comment]
Agreed about the name.  Also I'd prefer if the code supported the aura-like block time being slower than the babe slot time, although I know they'll use the same for now, so the original babe is just the new babe with the aura-like slot time set quite long.
